pkgs/servers/monitoring/nagios/default.nix

```diff
@@ -1,16 +1,16 @@
-{ stdenv, fetchurl, perl, php, gd, libpng, zlib }:
+{ stdenv, fetchurl, perl, php, gd, libpng, zlib, unzip }:
 
 stdenv.mkDerivation rec {
   name = "nagios-${version}";
-  version = "4.0.8";
+  version = "4.2.3";
 
   src = fetchurl {
     url = "mirror://sourceforge/nagios/nagios-4.x/${name}/${name}.tar.gz";
-    sha256 = "0jyad39wa318613awlnpczrrakvjcipz8qp1mdsig1cp1hjqs9lb";
+    sha256 = "0p16sm5pkbzf4py30hwzm38194cl23wfzsvkhk4jkf3p1fq7xvl3";
   };
 
   patches = [ ./nagios.patch ];
-  buildInputs = [ php perl gd libpng zlib ];
+  buildInputs = [ php perl gd libpng zlib unzip ];
```

Comment:
I do not think that unzip it is related to the CVE:

```bash
$ nix-build -A nagios
…
gcc -Wall -I.. -g -O2 -DHAVE_CONFIG_H -DNSCGI  -o summary.cgi summary.c ../common/shared.o getcgi.o cgiutils.o cgiauth.o macros-cgi.o  objects-cgi.o xobjects-cgi.o statusdata-cgi.o xstatusdata-cgi.o comments-cgi.o downtime-cgi.o ../lib/libnagios.a 
gcc -Wall -I.. -g -O2 -DHAVE_CONFIG_H -DNSCGI  -o tac.cgi tac.c ../common/shared.o getcgi.o cgiutils.o cgiauth.o macros-cgi.o  objects-cgi.o xobjects-cgi.o statusdata-cgi.o xstatusdata-cgi.o comments-cgi.o downtime-cgi.o ../lib/libnagios.a  
gcc -Wall -I.. -g -O2 -DHAVE_CONFIG_H -DNSCGI  -o statuswrl.cgi statuswrl.c ../common/shared.o getcgi.o cgiutils.o cgiauth.o macros-cgi.o  objects-cgi.o xobjects-cgi.o statusdata-cgi.o xstatusdata-cgi.o comments-cgi.o downtime-cgi.o ../lib/libnagios.a -lm 
gcc -Wall -I.. -g -O2 -DHAVE_CONFIG_H -DNSCGI  -o statusmap.cgi statusmap.c ../common/shared.o getcgi.o cgiutils.o cgiauth.o macros-cgi.o  objects-cgi.o xobjects-cgi.o statusdata-cgi.o xstatusdata-cgi.o comments-cgi.o downtime-cgi.o ../lib/libnagios.a -lgd  -lpng -ljpeg -lz -lm 
gcc -Wall -I.. -g -O2 -DHAVE_CONFIG_H -DNSCGI  -o trends.cgi trends.c ../common/shared.o getcgi.o cgiutils.o cgiauth.o macros-cgi.o  objects-cgi.o xobjects-cgi.o statusdata-cgi.o xstatusdata-cgi.o comments-cgi.o downtime-cgi.o ../lib/libnagios.a -lgd  -lpng -ljpeg -lz -lm 
gcc -Wall -I.. -g -O2 -DHAVE_CONFIG_H -DNSCGI  -o histogram.cgi histogram.c ../common/shared.o getcgi.o cgiutils.o cgiauth.o macros-cgi.o  objects-cgi.o xobjects-cgi.o statusdata-cgi.o xstatusdata-cgi.o comments-cgi.o downtime-cgi.o ../lib/libnagios.a -lgd  -lpng -ljpeg -lz -lm 
make[1]: Leaving directory '/tmp/nix-build-nagios-4.2.3.drv-0/nagios-4.2.3/cgi'
cd ./html && make
make[1]: Entering directory '/tmp/nix-build-nagios-4.2.3.drv-0/nagios-4.2.3/html'
(cd angularjs && unzip -u angular-1.3.9.zip)
/nix/store/mgdm9y2lhmyb74wwa42qhcl633vbwn26-bash-4.3-p46/bin/bash: unzip: command not found
make[1]: *** [Makefile:20: all] Error 127
make[1]: Leaving directory '/tmp/nix-build-nagios-4.2.3.drv-0/nagios-4.2.3/html'
make: *** [Makefile:75: all] Error 2
builder for ‘/nix/store/zisvfirpbs454rdskf92f31q02g3gv1b-nagios-4.2.3.drv’ failed with exit code 2
error: build of ‘/nix/store/zisvfirpbs454rdskf92f31q02g3gv1b-nagios-4.2.3.drv’ failed
```

This version seems to embed a web UI build on angularjs, and angular is embedded as a zip in the distribution. Not having it would mean that we would have to cherry-pick the security related updates from nagios-4.2.3 into nagios-4.0.8.